Sources/Misc/Codable/AnyEncodable.swift

```diff
@@ -109,4 +109,22 @@ extension AnyEncodable: Decodable {
 
 }
 
+extension AnyEncodable: Equatable {
+    static func == (lhs: AnyEncodable, rhs: AnyEncodable) -> Bool {
+        let lhsEncoder = JSONEncoder()
+        let rhsEncoder = JSONEncoder()
+
+        lhsEncoder.dateEncodingStrategy = .iso8601
+        rhsEncoder.dateEncodingStrategy = .iso8601
```

Comment:
we can simplify this bit by just doing something like 
```swift
        do {
            let lhsData = try lhs.encodedJSON
            let rhsData = try rhs.encodedJSON
            return lhsData == rhsData
            return lhsData == rhsData
        } catch {
            return false
        }
```

Reply:
so clean!